LookupTable.lua

```diff
@@ -95,6 +108,25 @@ function LookupTable:accGradParameters(input, gradOutput, scale)
    )
 end
 
+function LookupTable:renorm(input)
+   if not self.maxNorm then
+      return
+   end
+   local row_idx = input:clone()
+   if row_idx:dim() == 2 then
+      row_idx = row_idx:view(-1)
+   elseif row_idx:dim() ~= 1 then
+      error("input must be a vector or matrix")
+   end
+   -- "row_idx" and "weight" will be modified in the C code
+   self.weight.THNN.LookupTable_renorm(
+      row_idx:cdata(),
+      self.weight:cdata(),
+      self.maxNorm,
+      self.normValue or 2
```

Comment:
looks like maxNorm and normValue args are swapped compared to the C function definition below.